services/clinical/index.js

```diff
@@ -87,6 +90,19 @@ const getClinicalSubmissionSchemaVersion = async () => {
   return response.version;
 };
 
+/**
+ * @returns {Promise<boolean>}
+ */
```

Comment:
![image](https://i.kym-cdn.com/photos/images/original/001/240/434/827.gif)

Reply:
looks like github does have the emoji so think of this as the party parrot... :taco: